Assets/Scripts/Layers/Adapters/DataTypeAdapters/GeoJSONImportAdapter.cs

```diff
@@ -66,8 +70,14 @@ private void CreateGeoJSONLayer(LocalFile localFile, UnityEvent<string> onErrorC
             if (onErrorCallback != null)
                 newLayer.OnParseError.AddListener(onErrorCallback.Invoke);
 
-            
-            newLayer.SetURL(localFilePath, localFile.SourceUrl);
+            var fullPath = localFile.LocalFilePath;
```

Comment:
dit is niet de fullPath, maar de local path (Application.PersistentDatapath + fullPath is de absolute path). maar volgens mij heeft dit verder geen invloed op de rest